linode/domain/framework_schema_resource.go

```diff
@@ -0,0 +1,141 @@
+				stringplanmodifier.RequiresReplace(),
+			},
+		},
+		"group": schema.StringAttribute{
+			Description: "The group this Domain belongs to. This is for display purposes only.",
+			Optional:    true,
+			Validators: []validator.String{
+				stringvalidator.LengthBetween(0, 50),
+			},
+		},
+		"status": schema.StringAttribute{
+			Description: "Used to control whether this Domain is currently being rendered.",
+			Optional:    true,
+			Computed:    true,
+			Default:     stringdefault.StaticString("active"),
+		},
+		"description": schema.StringAttribute{
+			Description: "A description for this Domain. This is for display purposes only.",
+			Optional:    true,
+			Validators: []validator.String{
+				stringvalidator.LengthBetween(0, 253),
+			},
+		},
+		"master_ips": schema.SetAttribute{
+			ElementType: types.StringType,
+			Description: "The IP addresses representing the master DNS for this Domain.",
+			Optional:    true,
+			Computed:    true,
+			PlanModifiers: []planmodifier.Set{
+				setplanmodifier.UseStateForUnknown(),
+			},
+		},
+		"axfr_ips": schema.SetAttribute{
+			ElementType: types.StringType,
+			Description: "The list of IPs that may perform a zone transfer for this Domain. This is potentially " +
+				"dangerous, and should be set to an empty list unless you intend to use it.",
+			Optional: true,
+			Computed: true,
+			PlanModifiers: []planmodifier.Set{
+				setplanmodifier.UseStateForUnknown(),
+			},
+		},
+		"ttl_sec": schema.Int64Attribute{
+			Description: "'Time to Live' - the amount of time in seconds that this Domain's records may be " +
+				"cached by resolvers or other domain servers. " + domainSecondsDescription,
+			Optional:   true,
+			Computed:   true,
+			CustomType: customtypes.LinodeDomainSecondsType{},
+			PlanModifiers: []planmodifier.Int64{
+				int64planmodifier.UseStateForUnknown(),
+			},
+		},
+		"retry_sec": schema.Int64Attribute{
+			Description: "The interval, in seconds, at which a failed refresh should be retried. " +
+				domainSecondsDescription,
+			Optional:   true,
+			Computed:   true,
+			CustomType: customtypes.LinodeDomainSecondsType{},
+			PlanModifiers: []planmodifier.Int64{
+				int64planmodifier.UseStateForUnknown(),
+			},
+		},
+		"expire_sec": schema.Int64Attribute{
+			Description: "The amount of time in seconds that may pass before this Domain is no longer " +
+				domainSecondsDescription,
+			Optional:   true,
+			Computed:   true,
+			CustomType: customtypes.LinodeDomainSecondsType{},
+			PlanModifiers: []planmodifier.Int64{
+				int64planmodifier.UseStateForUnknown(),
+			},
+		},
+		"refresh_sec": schema.Int64Attribute{
+			Description: "The amount of time in seconds before this Domain should be refreshed. " +
+				domainSecondsDescription,
+			Optional:   true,
+			Computed:   true,
+			CustomType: customtypes.LinodeDomainSecondsType{},
+			PlanModifiers: []planmodifier.Int64{
+				int64planmodifier.UseStateForUnknown(),
+			},
+		},
```

Comment:
I don't think we should add `UseStateForUnknown()` to these attributes, because they're updatable in the Update plan. cc @zliang-akamai how do you think?

Reply:
@yec-akamai So `UseStateForUnknown()` would only be using the state values when the user doesn't explicitly configure these values in the tf file. I don't think these values can be updated during an update operation when the user didn't configure them. Am I correct?

Reply:
https://github.com/linode/terraform-provider-linode/pull/899/files#diff-f38fff6b6d425a04b761a731e00f7c4101b5361046353b365db5ba4be83aef71R198-R201
These fields are included in the updateOptions, so I think they can be updated in the config. 

> when the user didn't explicitly configure them.
I remember last time we discussed, if we set `UseStateForUnknown()` to an attribute, it will not be applied `known after apply`, even if it can be configured explicitly and it may be misleading to users. That's why I think we don't want to have `UseStateForUnknown()` to these fields because they can be configured.

Reply:
So my observations are:
- When the attribute is a **computed** attribute **and is not explicitly configured** by the user, it will show `known after apply` during `terraform plan`.
- For some computed attributes, TF provider will update them without explicitly configured by the user, e.g. `updated`. And we shouldn't apply `UseStateForUnknown()` for them because of misleading annotations (state values) instead of `known after apply` shown during the `terraform plan`.
- For some other (computed or not) attributes, TF provider will only update them when the user explicitly configured a new value. We should be safe to use `UseStateForUnknown()`. And I think these attributes belong to this case.

Did I miss anything? Or these attributes can be updated without explicitly configured new value?

Reply:
Gotcha. I thought `UseStateForUnknown()` will impact explicitly configured attributes as well. Now it's all good, thanks for the explanation